test/e2e/staking.js

```diff
@@ -50,106 +49,33 @@ test("staking", async function(t) {
+    // Select the second validator.
+    await app.client.click(`//*[. = 'local_2']`)
 
-    bondedStake += 20
+    // For some reason we need to sleep at this point in order to prevent the
+    // following error:
+    //
+    // Element <span class="tm-btn__container tm-btn--primary">...</span> is not
+    // clickable at point (960, 219). Other element would receive the click:
+    // <div class="ps__rail-y" style="top: 0px; height: 557px; right:
+    // 0px;">...</div>
+    await sleep(500)
 
-    // wait until the validators are showing again
-    await app.client.waitForVisible("#go-to-bonding-btn", 30000)
-    // TODO: re-enable once we've added back optimistic updates
-    // t.equal(
-    //   await app.client.$(".li-validator__value.your-votes").getText(),
-    //   bondedStake.toString(),
-    //   "Delegate steak in validator updated correctly"
-    // )
-
-    t.end()
-  })
-
-  t.test("unbonding", async function(t) {
-    // validator should already be in the cart so we only need to click a button to go to the bonding view
-    await app.client.$("#go-to-bonding-btn").click()
-    // the unbonded atoms are equal to the total owned stake minus the bonded atoms (150 - 100 == 50)
-    //TOD: re-enable after optimistic loading (or actual loading) update
-    // t.ok(
-    //   await waitForValue(
-    //     () => app.client.$("#new-unbonded-atoms"),
-    //     (totalUserStake - bondedStake).toString()
-    //   ),
-    //   "Left over steak shows correctly"
-    // )
-    // the bonded amount should show 120
-    // t.equal(
-    //   await app.client.$(".bond-candidate .bond-value__input").getValue(),
-    //   bondedStake.toString(),
-    //   "Candidate bond matches current bond"
-    // )
-
-    // the bonded candidate is lowered by 20 stake
-    // this is reflected correctly in the unbonded atoms section (50 - 20 === 30)
     await app.client
-      .$(".bond-candidate .bond-value__input")
-      .setValue(bondedStake - 20)
-    //TOD: re-enable after optimistic loading (or actual loading) update
-    // t.equal(
-    //   await app.client.$("#new-unbonding-atoms").getValue(),
-    //   (20).toString(),
-    //   "Unbonding steak shows correctly"
-    // )
-
-    await app.client.$("#btn-bond").click()
-
-    // should fail
-    t.ok(await app.client.isVisible(".tm-form-msg--error"), "shows error")
-
-    await app.client.$("#bond-confirm").click()
-    await app.client.$("#btn-bond").click()
-
-    t.ok(!(await app.client.isVisible(".tm-form-msg--error")), "hides error")
-
-    bondedStake -= 20
-
-    // wait until the validators are showing again
-    // await app.client.waitForVisible("#go-to-bonding-btn", 30000)
-    // TODO: Update when we fix the optimistic ui changes again
-    // t.equal(
-    //   await app.client.$(".li-validator__value.your-votes").getText(),
-    //   bondedStake.toString(),
-    //   "Delegate steak in validator updated correctly"
-    // )
-
+      .click(`//button/*[. = 'Stake']`)
```

Comment:
I have the feeling an id would be better here. What do you think=

Reply:
Sure, that's fine.  After I see your response to the comment above I can create another PR with it.

Reply:
Not a blocking concern for me.

Reply:
OK then I won't bother.